testhelpers/testhelpers.go

```diff
@@ -285,6 +262,14 @@ func RecursiveCopy(t *testing.T, src, dst string) {
 			AssertNil(t, err)
 			RecursiveCopy(t, filepath.Join(src, fi.Name()), filepath.Join(dst, fi.Name()))
 		}
+		if fi.Mode()&os.ModeSymlink != 0 {
```

Comment:
Now that we have this code in `RecursiveCopy` I wonder if this code is now redundant and could be removed: https://github.com/buildpacks/lifecycle/blob/63c5c6e6a2f5c6617781ed6d42a80b1c3918730b/testhelpers/binaries.go#L94-L105